templates/base_index.html

```diff
@@ -1,330 +1,1583 @@
+        </li>
+        <li class="p-list__item is-ticked">
+          Python Operator Framework for rapid development
+        </li>
+        <li class="p-list__item is-ticked">
+          Bring the K8s operator pattern to classic workloads
+        </li>
+        <li class="p-list__item is-ticked">
+          Lifecycle management and Day 2 operations
+        </li>
+        <li class="p-list__item is-ticked">
+          <a href="https://jaas.ai/store" class="p-link--external">Open collection</a> of community-driven operators
+        </li>
+      </ul>
+      <p>
+        <a class="p-link--external" href="https://juju.is">
+          Learn more about universal operators
+        </a>
+      </p>
+    </div>
+  </div>
+
+    <div clas="row">
+      <ul class="p-inline-images--small u-no-margin--bottom">
+        <li class="p-inline-images__item">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/1f395c09-Panasonic-Logo.svg",
+            alt="Panasonic logo",
+            width="130",
+            height="21",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-inline-images__logo is-wide"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/7bbfb1d0-Scania_logo.svg",
+            alt="Scania logo",
+            width="153",
+            height="40",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-inline-images__logo is-wide"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/b41ec3f5-SBI-Bits_logo.png",
+            alt="SBI Bits logo",
+            width="132",
+            height="40",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item">
+          {{
+            image(
+            url="https://assets.ubuntu.com/v1/890ffa71-Swissquote_logo.svg",
+            alt="Swissquote logo",
+            width="210",
+            height="40",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-inline-images__logo is-wide"},
+            ) | safe
+          }}
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+{% block footer_content %}
+{# include "ubuntu/takeovers/_footer_homepage.html" #}
+{% endblock footer_content %}
+
+<script>
+  function getPrimaryParentLanguage() {
+    /**
+    * Get the primary parent language
+    *
+    * Get the user's default language, and if it's a sub-language,
+    * infer the parent language - e.g. for "en-GB", infer "en"
+    */
+    var language = navigator.language || navigator.userLanguage || navigator.browserLanguage || navigator.systemLanguage;
```

Comment:
```suggestion
    let language = navigator.language || navigator.userLanguage || navigator.browserLanguage || navigator.systemLanguage;
```

Reply:
Let doesn't work in older browers and this is not compiled.

Reply:
We supporting IE10? https://caniuse.com/#feat=let